src/utils/trusts.ts

```diff
@@ -30,6 +30,13 @@ const checkEntityRequiresTrusts = (appData: ApplicationData | undefined): boolea
     containsTrusteeNatureOfControl(appData.beneficial_owners_corporate ?? []);
 };
 
+const checkEntityRequiresManageTrusts = (appData: ApplicationData | undefined): boolean => {
+  if (!appData?.update?.review_trusts) {
```

Comment:
did we have a one liner for this?

Reply:
just following up on this, Joshua had responded in the initial PR saying it was complaining of undefined so this was best approach.

https://github.com/companieshouse/overseas-entities-web/pull/1047/files#r1324152113